org.civicrm.reqangular/src/common/services/format-currency.service.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/services',
+  'common/services/api',
+  'common/models/settings.model'
+], function (services) {
+  'use strict';
+
+  services.factory('FormatCurrencyService', FormatCurrencyService);
+
+  FormatCurrencyService.$inject = ['$log', '$q', 'Settings'];
+
+  function FormatCurrencyService ($log, $q, Settings) {
+    $log.debug('Service: FormatCurrencyService');
+
+    return {
+      format: format
+    };
+
+    /**
+    * Adds defined thousand and decimal separators in the given amount
+    *
+    * @param {string} amount
+    * @return {promise}
```

Comment:
`{Promise}`

Reply:
well, to be honest all need to be capitalised since they are all classes. even the {String}.

![image](https://user-images.githubusercontent.com/3973243/32272571-8a109338-bef6-11e7-8d77-aaed3b3a9976.png)

cc @bmnepali @AkA84